.github/workflows/ci.yml

```diff
@@ -205,3 +213,8 @@ jobs:
             checkout/${{ matrix.app }}/Saved/Stats
             checkout/${{ matrix.app }}/Saved/MaterialStats
             checkout/${{ matrix.app }}/Saved/MaterialStatsDebug
+
+      - name: Run tests
+        if: ${{ matrix.app }}
+        run: |
+          docker exec -w /workspace/checkout/${{ matrix.app }} unreal ue4 test --filter Product
```

Comment:
@tustanivsky this breaks on the following test - I didn't spot an obvious issue in the code but I'm far from certain about all the pointers involved

```
[2022.08.31-20.01.20:509][149]LogAutomationController: Display: Test Started. Name={should affect events to be captured}
[2022.08.31-20.01.20:509][149]LogAutomationController: Sending RunTest should affect events to be captured to 9E98CC2AEB804E09B8868F8CD8934762
[2022.08.31-20.01.20:521][150]LogAutomationWorker: Received RunTests should affect events to be captured from F6E0196FB1694DBBB7938F61AC029B25

[2022.08.31-20.01.20:552][151]LogCore: === Critical error: ===
Unhandled Exception: SIGSEGV: invalid attempt to read memory at address 0x0000000000000010

[2022.08.31-20.01.20:552][151]LogCore: Fatal error!

0x00007f914cf3ab9d libsentry.so!sentry_value_get_by_key(+0x5d)
0x00007f914cfcf371 libUE4Editor-Sentry.so!SentryEventDesktop::GetLevel() const [/workspace/checkout/sample/Plugins/sentry/Source/Sentry/Private/Desktop/SentryEventDesktop.cpp:47]
0x00007f914cfda53e libUE4Editor-Sentry.so!UE4Function_Private::TFunctionRefCaller<SentrySubsystemSpec::Define()::$_20::operator()() const::'lambda'(), void ()>::Call(void*) [/home/ue4/UnrealEngine/Engine/Source/Runtime/Core/Public/Templates/Function.h:548]
0x00007f91cc45dda8 libUE4Editor-BuildPatchServices.so!UnknownFunction(0x312da7)
0x00007f91da4e0fc5 libUE4Editor-Core.so!_ZN24FAutomationTestFramework21ExecuteLatentCommandsEv(+0x124)
0x00007f9172ff65e6 libUE4Editor-AutomationWorker.so!UnknownFunction(0x135e5)
0x000000000023a4c7 UE4Editor!UnknownFunction(0x3a4c6)
0x0000000000246eca UE4Editor!UnknownFunction(0x46ec9)
0x00007f91daf53056 libUE4Editor-UnixCommonStartup.so!_Z14CommonUnixMainiPPcPFiPKDsEPFvvE(+0x845)
0x00007f91d0e16bf7 libc.so.6!__libc_start_main(+0xe6)
0x00000000002304a9 UE4Editor!UnknownFunction(0x304a8)